_pages/locations/washington-dc.md

```diff
@@ -164,43 +173,43 @@ There are maps of the D.C. office in every elevator lobby.
 
 Gendered bathrooms are located at the end of the 3100 wing. A single-occupancy, wheelchair-accessible all-gender bathroom is located on the third floor in wing three (room 3355).
 
-### Mother’s rooms
+### Nursing/Pumping rooms
 
-Look for room 1232A at the back of the health center on the first floor. You don’t need to reserve it. If it’s in use, you can use the doctor’s office or one of the other rooms near the health clinic. After 4 p.m., you can access the mother’s room directly from the hallway, but the rest of the health center will be closed.
+Look for room 1232A at the back of the health center on the first floor. You don’t need to reserve it. If it’s in use, you can use the doctor’s office or one of the other rooms near the health clinic. After 4 p.m., you can access the nursing/pumping room directly from the hallway, but the rest of the health center will be closed.
 
-There is a fridge inside the mother’s room and one outside the room as well. At the end of the day, if there’s any milk in the outer fridge, they’ll move it to the inner fridge.
+There is a fridge inside the room and one outside the room, in the health clinic. At the end of the day, if there’s any milk in the outer fridge, they’ll move it to the inner fridge.
 
 ### Yoga and meditation rooms
 
 Room 3042, at the corner of the 3000 and 3300 wings, is available for yoga and meditation/mindfulness/prayer.
 
 ### Batteries
 
-If you need batteries for your keyboard or trackpad, there is a consolidated supply closet on the first floor of the building, in the 200 wing. There are also rechargeable batteries in a charger plugged into one of the columns in the 4462 area.
+If you need batteries for your keyboard or trackpad, there is a consolidated supply closet on the first floor of the building, in the 200 wing. 
+
+### Spare keyboards and mice
+
+Melvin Moreira in GSA IT has all of the spare Mac equipment. You can email him to confirm that he is in the office. He sits on the ground floor, wing zero, in room G035.
```

Comment:
The handbook reviewers are working out a new rule that no one person should be named in documentation. Granted Melvin is amazing but he is also a GSA IT contractor and as such the proper means to requesting spare equipment is a Servicedesk ticket.